File: babelfish-solution.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);

	    Map<String,String> words = new HashMap<>();

	    while(true){
	    	String input = sc.nextLine();

	    	if(input.equals(""))break;

	    	String[] split_input = input.split(" ");

	    	String key = split_input[1];
	    	String value = split_input[0];

	    	words.put(key,value);
		}

		try{
	    	while(true){
	    		String input = sc.nextLine();

	    		String key = "eh";

	    		if(words.containsKey(input))key = words.get(input);

	    		System.out.println(key);
			}
		}catch(Exception e){

		}
    }
}

File: babelfish-solution.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Solution
{
    class Program
    {
        static void Main(string[] args)
        {
            Dictionary<string, string> words = new Dictionary<string, string>();
            string input;

            while (true)
            {
                input = Console.ReadLine();

                if (input == "") break;

                string[] split_input = input.Split();

                string key = split_input[1];
                string value = split_input[0];

                words.Add(key, value);
            }

            try
            {
                while (true)
                {
                    input = Console.ReadLine();
                    string key = "eh";

                    if (words.ContainsKey(input)) key = words[input];

                    Console.WriteLine(key);
                }
            }
            catch
            {

            }
        }
    }
}
